### PR TITLE
 beman.execution26: change library status to under development

### PR DESCRIPTION
**Pull request by RaduNichita**
Issues: https://github.com/bemanproject/beman/issues/77

Expected README.md:

image

### CODE REVIEW

**Review by neatudarius — APPROVED** [review]
I agree the API is stable (it's in the Draft). I ask @dietmarkuehl to confirm if the library is also production ready. 

Thanks!

CC: @bemanproject/leads

File: README.md [hunk 1]
```diff
@@ -33,7 +35,9 @@ e.g.:
     completed.
 - `bulk(...)` to executed execute work, potentially concurrently.
 
-**Implements:** [`std::execution` (P2300)](http://wg21.link/p2300).
+**Implements:** [`std::execution` (P2300R10)](http://wg21.link/P2300R10).
+
+**Status**: [Production ready. Stable API.](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md#production-ready-stable-api)
```
Comment (dietmarkuehl) [inline README.md:40]: I'm not convinced that is the correct status: there are few bits missing from P2300 which still need to be implemented. There are also a number of things tagged "TODO" in the code which should be addressed in some form.

Comment (neatudarius) [reply README.md:40]: Ok, this is why we explicitly asked you. What you recommend? (Please check BEMAN_LIBRARY_MATURITY_MODEL.md)

Comment (dietmarkuehl) [reply README.md:40]: Looking at the [Beman Library Maturity Model](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md) the implementation is "under development": while the API is approved and merged into the working paper, the implementation isn't complete covering all of the API. Also, the API isn't standardised, yet, an possibly subject to change.

One interesting side question is: the repo currently contains [partial] implementations of multiple papers:

- [std::execution](https://wg21.link/P2300) which is merged into the standard.
- [async_scope -- Creating scopes for non-sequential concurrency](https://wg21.link/P3149) which is moving towards getting merged soon.
- [C++ Concurrent Queues](https://wg21.link/P0260) which is under discussion and may get merged soon.

There are also a few additional papers in the space which fit the theme of this project:

- [finally, write_env, and unstoppable Sender Adaptors](https://wg21.link/P3284).
- [let_with_async_scope](https://wg21.link/P3296).
- [Allocator Support for Operation States](https://wg21.link/P3433), already implemented.
- [Enabling more efficient stop-token based cancellation of senders](https://wg21.link/P3409), partially implemented, although rejected. I think some of that could be salvaged when opting into use of a constrained stop token.
- [Reducing operation-state sizes for subobject child operations](https://P3425).
- there are possibly more

For some of these proposal it doesn't make any sense to got into a separate Beman project as the inherently change how things are implemented. Others are add-ons which may get into the same header. Creating them in a separate Beman project would, at minimum require some approach to reconcile how they are used. Some are just add-ons and could get into a separate Beman project requiring separate infrastructure and creating a more awkward user experience. Which raises a few questions:

1. Should things be factored into different Beman projects if it is possible?
2. For things within one project, the proposals may be in different state for readiness: should they have separate maturity indicators?

I guess, that may be a discussion for Discourse and I'll post the question there.

**Review by dietmarkuehl — CHANGES_REQUESTED** [review]
The status needs to be adjusted: the library is currently still under development

**Review by neatudarius — APPROVED** [review]

**Review by dietmarkuehl — APPROVED** [review]

**Pull request merged**